po/de.po

```diff
@@ -14091,7 +14091,7 @@ msgstr "Beispiel:"
 
 #: ../src/gnome-utils/gtkbuilder/dialog-preferences.glade.h:48
 msgid "<b>Account Color</b>"
-msgstr "<b>_Kontofarbe</b>"
```

Comment:
This is one of many dialog labels, where the source is missing the mnenomic ("_").
Currently I am planning to set the attribute `<property name="use_underline">True</property>`
in the glade files. Then we can use them already in the translation.

If the MsgIds should be changed now - breaking the string freeze - or later should be discussed the next days on gnucash-devel.

Reply:
Hi, I'm happy to be back :-)

It seems in this case like a mistake in german translation:
![bildschirmfoto_2018-01-06_10-59-17_kontofarbe](https://user-images.githubusercontent.com/426910/34638676-3dd112ae-f2d1-11e7-9d00-ca87e6dd4c9f.png)

Reply:
Right, instead the next 2 entries should get some.